### ISSUE TITLE
Dropdown menu is not accessible

### ISSUE THREAD

**Issue by sarahbilly** [Baremetrics/calendar#106]
When there is a dropdown option to select date ranges, it's only available with mouse click but a keyboard or screenreader user cannot access this, since it's just using `div` and not `button` (or using `role="button"` and adding in `tabindex="0"` and key press events). 

The other problem is that the dropdown options also need to be keyboard accessible. Would also need a button element.

**Comment by pieterbeulque** [comment]
Hey there, thanks for reaching out.

You're definitely 100% right, and we aim to make everything that gets out the door nowadays as accessible and inclusive as possible. However, we don't use the calendar / date picker in this repository ourselves anymore, so we won't maintain or update it any longer.

Feel free to open a PR with the required changes and I'd be happy to review and eventually merge them.
